meta-mender-core/recipes-mender/mender-client/mender-client-cpp.inc

```diff
@@ -14,4 +16,7 @@ FILES:${PN} += "\
     ${systemd_unitdir}/system/mender-authd.service \
 "
 
-inherit autotools
+# Don't download the googletest source in Mender CMakefiles.txt
+EXTRA_OECMAKE:append = " -DMENDER_DOWNLOAD_GTEST=NO"
```

Comment:
Even though we save bandwidth, this feels a bit hacky, especially since the license is also affected by it. Why can't we just clone it, even if we don't use it?

Reply:
It is because Yocto does not allow (at least without some hacking) downloading during anything but the `fetch` stage. 

I changed the client CMake build to accept this option, and instead rely on the pre-provided `googletest` recipe.

I would suspect this is not a problem?

Reply:
Ok, that makes sense. But then it probably should be downloaded in the fetch step, so that the license check still passes, or? It might require some extra parameters to the `SRC_URI` string.

Reply:
The recipe has a build dependency on `googletest`. Is that not enough?

Reply:
Oh, I see what you mean.

Remember, we don't vendor googletest anylonger

Reply:
You're right, I missed that part. Ignore my ranting then!